games.zetsubou.WanderNoMore.yml

```diff
@@ -0,0 +1,35 @@
+app-id: games.zetsubou.WanderNoMore
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: WanderNoMore.sh
+tags:
+  - proprietary
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --persist=.renpy
+modules:
+  - name: hmp
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin/
+      - cp -R * /app/bin/
+      - install -Dm644 -t "/app/share/metainfo/" games.zetsubou.WanderNoMore.metainfo.xml
+      - install -Dm644 -t "/app/share/applications/" games.zetsubou.WanderNoMore.desktop
+      - install -Dm644 games.zetsubou.WanderNoMore.png "/app/share/icons/hicolor/128x128/apps/games.zetsubou.WanderNoMore.png"
+    sources:
+      - type: archive
+        url: https://ftp.zetsubou.games/WanderNoMore/builds/v1.2.1.tar.bz2
```

Comment:
this contain x86_64 binaries but you still build an aarch64 flatpak as well.

also that would be nice to remove the i686 binaries

Did you actually try it? I see `python-2.7` and there is not Python 2.7 in the runtime.

Reply:
I've now added flathub.json to skip aarch64.

I have tried running this, including the build the bot created. It works for me on an ubuntu 22.04 system without python2 installed.
Although python is used, I believe the game ships with what it needs to run in the libs/python2.7 directory and doesn't require python2.7 to be installed on the host system.
If there's a particular operating system or environment these builds should be tested on I'm happy to spin up a virtual machine to make sure it works on there.

I can provide a new build with only x64 shortly.

Reply:
what's on the host is irrelevant.

Reply:
but if these python2.7 are not neded, you should remove them. There is `cleanup` for this.

Reply:
The libs in the python2.7 directory are needed.
In libs/linux-x86_64 is a python executable which runs them.
The engine used to run this game bundles the python binary and a minimal set of libraries in libs/python2.7 needed to run the game.

Reply:
ah ok.